### ISSUE TITLE
The third column of Jacobian is wrong

### ISSUE THREAD

**Issue by ss8es100111** [libnifalcon/libnifalcon#66]
The third column of Jacobian should be sin(theta2i)*sin(theta3i)/den, not sin(theta2i)*sin(theta2i)/den

**Comment by qdot** [comment]
Oh interesting! Nothing like 14 year old bugs. :| Any idea how much this improves our solution?

**Comment by ss8es100111** [comment]
I changed the Jacobian matrix, but it seems that there is no difference, haha
By the way, this website is dead (http://libnifalcon.nonpolynomial.com/), I can't see more information.

**Comment by qdot** [comment]
Yeah, this project hasn't been updated in at least a decade if not more and the project died a long time ago (it's still available on archive.org tho), though I still have like 4 Novint Falcons. Plan is to, at some point, rewrite it in bare C and reverse out the kinematics algos from the old Novint software, since the product is very dead at this point.
